connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/MirrorConnectorsIntegrationTransactionsTest.java

```diff
@@ -31,6 +32,7 @@
  * Integration test for MirrorMaker2 in which source records are emitted with a transactional producer,
  * which interleaves transaction commit messages into the source topic which are not propagated downstream.
  */
+@Tag("integration")
```

Comment:
Isn't this inherited already?

Reply:
It is, so this change is actually a no-op, I didn't realize that.
It is more consistent with the other MM2 integration tests which also inherit the tag.